pkg/api/authorizationschema/authorizationschema.swagger.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
in another PR, generated code might be even removed from version control

Reply:
hm, why not

Reply:
it can be left, it's not an issue